plugins/polio/tests/api/test.py

```
@@ -0,0 +1,63 @@
+            source_ref="PvtAI4RUMkr",
+        )
+        campaign = pm.Campaign.objects.create(
+            obr_name=obr_name,
+            country=country,
+            account=account,
+            vacine=pm.VACCINES[0][0],
+            separate_scopes_per_round=False,
+        )
+        scope_group = m.Group.objects.create(name="campaign_scope", source_version=source_version)
+        scope_group.org_units.set([district])  # FIXME: we should actually have children org units
+        scope = pm.CampaignScope.objects.create(campaign=campaign, vaccine=pm.VACCINES[0][0], group=scope_group)
```

Ruff (0.8.0)

Local variable `scope` is assigned to but never used

Remove assignment to unused variable `scope`

(F841)